src/main/java/Duke.java

```diff
@@ -1,10 +1,140 @@
+        } catch (DukeException ex) {
+            System.out.println(ex.getMessage());
+        }
+    }
+
+    public static void createDeadline(String userTxt, ArrayList<Task> listArr) {
+        String test = userTxt.substring(8);
+        try {
+            if (test.strip().equals("")) {
+                throw new DukeTaskNoDescException();
+            } else {
+                int splitIdx = userTxt.lastIndexOf("/");
+                String desc = userTxt.substring(9, splitIdx - 1);
+                String by = userTxt.substring(splitIdx + 3);
+                Task t = new Deadline(desc, by);
+                listArr.add(t); // add task to list
+                System.out.println("Got it. I've added this task:");
+                System.out.println(t);
+                System.out.println("Now you have " + listArr.size() + " tasks in the list.");
+            }
+        } catch (DukeException ex) {
+            System.out.println(ex.getMessage());
+        }
+    }
+
+    public static void createEvent(String userTxt, ArrayList<Task> listArr) {
+        String test = userTxt.substring(5);
+        try {
+            if (test.strip().equals("")) {
+                throw new DukeTaskNoDescException();
+            } else {
+                int frmIdx = userTxt.lastIndexOf(" /from");
+                int toIdx = userTxt.lastIndexOf("/to");
+                String desc = userTxt.substring(6, frmIdx);
+                String frm = userTxt.substring(frmIdx + 6, toIdx);
+                String to = userTxt.substring(toIdx + 3);
+                Task t = new Event(desc, frm, to);
+                listArr.add(t); // add task to list
+                System.out.println("Got it. I've added this task:");
+                System.out.println(t);
+                System.out.println("Now you have " + listArr.size() + " tasks in the list.");
+            }
+        } catch (DukeException ex) {
+            System.out.println(ex.getMessage());
+        }
+    }
+
+    public static void deleteTask(ArrayList<Task> listArr, int taskNum) {
+        listArr.get(taskNum);
+        System.out.println("Noted. I've removed this task:");
+        System.out.println(listArr.get(taskNum));
+        listArr.remove(taskNum);
+        System.out.println("Now you have " + listArr.size() + " tasks in the list.");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Jojo :)");
+        System.out.println("What can I do for you?");
+        System.out.println("-----------------------------------------------------------------------");
+        System.out.println("-----------------------------------------------------------------------");
```

Comment:
Maybe these few lines could be neater if you put all the strings into one `println` call? Like:
```
System.out.println("Hello...\n"
        + "What... \n"
        +...
);
````